CMEW/app/configure_for/bin/test_update_recipe_file.py

```diff
@@ -38,6 +47,12 @@ def path_to_mock_original_recipe():
 def test_update_recipe(
     mock_env_vars, path_to_updated_recipe_kgo, path_to_mock_original_recipe
 ):
+    """update_recipe should produce the KGO with both datasets updated.
+
+    - Dataset[0] uses REF_MODEL_ID / REF_VARIANT_LABEL
+    - Dataset[1] uses MODEL_ID / VARIANT_LABEL and ESMVal / amip settings
```

Comment:
```suggestion
    - Dataset[1] uses MODEL_ID / VARIANT_LABEL
```
Both datasets use ESMVal & amip, so not specific to dataset[1], I think. I suggest restricting this docstring to the variables defined in `mock_env_vars`.